src/rofl/common/crofctl.cc

```diff
@@ -2557,20 +2557,20 @@ void crofctl::send_packet_in_message(const cauxid &auxid, uint32_t buffer_id,
     switch (rofchan.get_version()) {
     case rofl::openflow12::OFP_VERSION: {
       if (is_slave()) {
-        return;
+        return rofl::crofsock::MSG_QUEUED;
```

Comment:
Why does it return `MSG_QUEUED` when no msg has been queued? (Many other places where this occurs)